src/whisk.rs

```diff
@@ -45,6 +45,13 @@ pub struct TrackerProof {
     s: Fr,
 }
 
+/// Convenience wrapper for whisk verifiers
+#[derive(Clone, Debug, CanonicalSerialize, CanonicalDeserialize)]
+pub struct ShuffleProof {
```

Comment:
How about we call it `WhiskShuffleProof` to make it clearer that it's a thing tailored to Whisk?